cypress/e2e/tests/migration/dynamic-reports/filter.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
```

Comment:
folder name should be same as in model . dynamicreports at both places or dynamic_reports

Reply:
Named both in models and tests as 'issues'